[2024-03-15 18:35:49,251] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.compute_rf_score 2024-03-15T18:34:00+00:00 [queued]>
[2024-03-15 18:35:49,289] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.compute_rf_score 2024-03-15T18:34:00+00:00 [queued]>
[2024-03-15 18:35:49,289] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2024-03-15 18:35:49,289] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2024-03-15 18:35:49,289] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2024-03-15 18:35:49,319] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): compute_rf_score> on 2024-03-15T18:34:00+00:00
[2024-03-15 18:35:49,331] {standard_task_runner.py:52} INFO - Started process 47631 to run task
[2024-03-15 18:35:49,339] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', '***_exam', 'compute_rf_score', '2024-03-15T18:34:00+00:00', '--job-id', '3400', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmp0o5h54dl', '--error-file', '/tmp/tmpn7946ku1']
[2024-03-15 18:35:49,340] {standard_task_runner.py:77} INFO - Job 3400: Subtask compute_rf_score
[2024-03-15 18:35:49,442] {logging_mixin.py:104} INFO - Running <TaskInstance: ***_exam.compute_rf_score 2024-03-15T18:34:00+00:00 [running]> on host 3a3d0401ac26
[2024-03-15 18:35:49,564] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=***_exam
AIRFLOW_CTX_TASK_ID=compute_rf_score
AIRFLOW_CTX_EXECUTION_DATE=2024-03-15T18:34:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-03-15T18:34:00+00:00
[2024-03-15 18:35:52,419] {python.py:151} INFO - Done. Returned value was: None
[2024-03-15 18:35:52,447] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=***_exam, task_id=compute_rf_score, execution_date=20240315T183400, start_date=20240315T183549, end_date=20240315T183552
[2024-03-15 18:35:52,550] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-03-15 18:35:52,605] {local_task_job.py:151} INFO - Task exited with return code 0
